server/http.go

```diff
@@ -72,6 +73,21 @@ func (p *Plugin) connectUserToZoom(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(state) == 0 {
```

Comment:
Nit pick: It idiomatic to write
```suggestion
	if state == "" {
```